Makefile

```
@@ -31,6 +31,8 @@ PIPECONF_OAR_FILE := $(DIR)/target/fabric-tna-1.2.0-SNAPSHOT.oar
 # Profiles to build by default (all)
 PROFILES ?= fabric fabric-upf fabric-int fabric-upf-int
 
+SHOW_SENSITIVE_OUTPUT ?= true
```

If you do export here you can remove it from the individual make targets.
```suggestion
export SHOW_SENSITIVE_OUTPUT ?= true
```

I tried this Makefile
```
export FOO ?= bar
echo:
	echo ${FOO}
```

And it works:
```
$ make echo
echo bar
bar
```